05-FITS/io_fits_ascii_overview.ipynb

```diff
@@ -646,9 +644,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.plot(throughput_data['lambda'], throughput_data['throughput'])\n",
-    "plt.xlabel('Wavelength ($\\mu$m)')\n",
-    "plt.ylabel('Throughput')"
+    "from astropy.visualization import quantity_support\n",
```

Comment:
Another good use of astropy.visualization (and a new feature I have haven't used before).